compose/compose-file/index.md

```diff
@@ -882,6 +882,81 @@ port (a random host port will be chosen).
      - "127.0.0.1:5000-5010:5000-5010"
      - "6060:6060/udp"
 
+### secrets
+
+After you
+[define or reference secrets in the top-level `secrets` configuration](#secrets-configuration-reference),
+grant access to secrets on a per-service basis using the per-service `secrets`
+configuration. Two different syntax variants are supported.
```

Comment:
I'm not on-board with this distinction, because the compose file reference is literally the only source for a lot of this stuff. This and the generic Secrets topic are all that people get, and this is a different way of thinking about configuring secrets than doing it all by hand.

Also, "secrets" can't allow you to do anything. "secrets" are not an actor. You are the actor, using a tool. So I don't think we should artifically use passive voice here. Can you unpick why this bothers you?